workspace-service/src/graphql/file_download_urls.rs

```diff
@@ -13,8 +14,25 @@ impl FileDownloadUrlsMutation {
     async fn file_download_url(&self, context: &Context<'_>, id: ID) -> FieldResult<Url> {
         let pool = context.data()?;
         let config = context.data()?;
+        let event_client: &EventClient = context.data()?;
+        let requesting_user = context.data::<super::RequestingUser>()?;
```

Comment:
For consistency you might want to put the type behind the variable everywhere.
```suggestion
        let requesting_user: &super::RequestingUser = context.data()?;
```